app/routes.js

```diff
@@ -1,14 +1,13 @@
 // @flow
 import React from 'react';
-import { Route, IndexRoute } from 'react-router';
+import { Route } from 'react-router';
 import App from './containers/App';
 import HomePage from './containers/HomePage';
 import CounterPage from './containers/CounterPage';
 
-
 export default (
   <Route path="/" component={App}>
-    <IndexRoute component={HomePage} />
+    <Route path="/" component={HomePage} />
```

Comment:
hi, thanks for the PR! I am playing around with it, but this did give me to following warning (also Homepage didn't show)
```
Warning: You should not use <Route component> and <Route children> in the same route; <Route children> will be ignored
```

I could get around it by removing the path compoent on the root route and using <App> directly. As seen in the first code example here https://teamtreehouse.com/community/warning-you-should-not-use-route-component-and-route-children-in-the-same-route-route-children-will-be-ignored